src/client/ui/select-element.js

```diff
@@ -265,3 +265,15 @@ export function switchOptionsByKeys (element, command) {
         }
     }
 }
+
+export function isOptionElementVisible (el) {
```

Comment:
look at the comment: https://github.com/DevExpress/testcafe/blob/master/src/client/ui/select-element.js#L1
To know is the `<option>` element visible or not we should know is the parent `<select>` element is expanded. During execution we "expand" this element via [testCafeUI function](https://github.com/DevExpress/testcafe/blob/master/src/client/ui/select-element.js#L121), so we can get this info from UI only.
We can refactor all this but it will affect on legacy and the new code and will take some additional time